range_generator.py

```diff
@@ -0,0 +1,14 @@
+def MyRange(start, stop, step):
+    counter = start
+    while counter < (stop):
+        yield counter
+        counter += step
+
+def my_range(start, stop, step):
```

Comment:
Why do we need this one? Is it supposed to be used/reviewed?

Reply:
I use this function to call "next" as many time as needed. So, I call the function my_range and it creates an object of the class MyRange, calling "next". Can we avoid using the extra function for this purpose?